pkg/diagnostics/cluster/roles.go

```diff
@@ -20,7 +20,34 @@ type ClusterRoles struct {
 }
 
 const (
-	ClusterRolesName = "ClusterRoles"
+	ClusterRolesName   = "ClusterRoles"
+	clusterRoleMissing = `
+clusterrole/%s is missing.
+
+Use the 'oadm policy reconcile-cluster-roles' command to create the role. For example,
+
+  $ oadm policy reconcile-cluster-roles \
```

Comment:
I don't see where it explicitly says to not use dollar signs but I think this example is subtly different.  Here we are suggesting a command you can run to resolve a discovered issue.  We are not suggesting an example of running a diagnostics command.  Regardless, I suggest we either add '$' or remove them to be consistent with places we have used them before.